src/gui/editors/PianoRoll.cpp

```diff
@@ -202,10 +202,10 @@ PianoRoll::PianoRoll() :
 	signalMapper = new QSignalMapper( this );
 	m_semiToneMarkerMenu = new QMenu( this );
 
-	QAction * markSemitoneAction = new QAction( tr("Mark/unmark current semitone"), this );
-	QAction* markScaleAction = new QAction( tr("Mark current scale"), this );
-	QAction* markChordAction = new QAction( tr("Mark current chord"), this );
-	QAction* unmarkAllAction = new QAction( tr("Unmark all"), this );
+	QAction *markSemitoneAction = new QAction( tr("Mark/unmark current semitone"), this );
+	QAction *markScaleAction = new QAction( tr("Mark current scale"), this );
+	QAction *markChordAction = new QAction( tr("Mark current chord"), this );
+	QAction *unmarkAllAction = new QAction( tr("Unmark all"), this );
```

Comment:
Well, even if I like `char* a` more I have to agree that `char *a` removes ambiguity, so that's what I would agree to be a standard. Any of those two is much better than the commonly seen `char * a` on lmms codebase.